graphql/database/users/deleteUser.js

```diff
@@ -0,0 +1,48 @@
+import uuid from 'uuid/v4'
+import UserModel from './UserModel'
+import UserWidgetModel from '../widgets/userWidget/UserWidgetModel'
+import getWidgets from '../widgets/getWidgets'
+
+/**
+ * Delete a user and performs other cleanup actions.
+ * This function must be idempotent, as the client may call it
+ * for existing users.
```

Comment:
Remove: "This function must be idempotent, as the client may call it for existing users."

Maybe more precisely: "Delete all personally identifiable information, sensitive, and potentially sensitive user data."